ArduPlane/quadplane.cpp

```diff
@@ -1896,7 +1908,7 @@ void QuadPlane::update_throttle_suppression(void)
     }
 
     // if the users throttle is above zero then allow motors to run
-    if (plane.get_throttle_input() != 0) {
+    if ((plane.get_throttle_input() != 0) || (plane.air_mode == AirMode::ON)) {
```

Comment:
should this be on all flight modes?

Reply:
changed to apply only to non-auto throttle modes